addons/l10n_es/migrations/12.0.4.0/openupgrade_analysis.txt

```diff
@@ -2,3 +2,4 @@
 ---Fields in module 'l10n_es'---
 ---XML records in module 'l10n_es'---
 DEL account.account.template: l10n_es.account_common_572999
+DEL account.tax.group: l10n_es.tax_group_iva_10-5 (noupdate)
```

Comment:
This appears because the improvement in v11 has not been forwardported yet to v12.

Reply:
Well, maybe it's worth to wait for it.

Reply:
Yeah, I will wait for it.

Reply:
Resolved.